.gitignore

```diff
@@ -117,6 +117,7 @@ tools/*.dll
 
 # Sensitive files
 *.keys
+!Azure.AspNetCore.DataProtection.Keys
```

Comment:
Do we want to leave this in?

Reply:
Yes, otherwise the project directory gets ignored by `*.keys` wildcard. We did the same thing for KV.Keys one line bellow.